spec/models/product_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Product, type: :model do
+  describe '\'Products\' has db columns' do
+    it {is_expected.to have_db_column :id}
```

Comment:
You don't need to test for the ID, every instance of a database table will get an ID because it is the primary key, so it is unnecessary to test for. It happens wether we want it or not.

Reply:
removed test for id and added one for description